models/rnn.py

```diff
@@ -0,0 +1,132 @@
+import numpy as np
+import theano.tensor as T
+from theano import scan
+from theano.ifelse import ifelse
+
+from smartpy import Model
+from smartpy.misc.utils import sharedX
+from .utils import WeightsInitializer
+
```

Comment:
From the Theano documentation I think it is preferable to use MRG_RandomStreams
`from theano.sandbox.rng_mrg import MRG_RandomStreams as RandomStreams`